modules/manta/germline/main.nf

```diff
@@ -8,11 +8,12 @@ process MANTA_GERMLINE {
         'quay.io/biocontainers/manta:1.6.0--h9ee0642_1' }"
 
     input:
-    tuple val(meta), path(input), path(input_index)
+    tuple val(meta), path(input)
```

Comment:
I wuld propose to leave the index with the input. We have been going back and forth on this multiple times. This just avoids needless merging and splitting of channels

Reply:
Sure! I will fix that 👍🏻